internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -396,11 +401,19 @@ func metadataToDocument(metadata *metadata) *types.Document {
 			keyDoc.Set(pair.Field, int32(pair.Order)) // order is set as int32 to be sjson-marshaled correctly
 		}
 
+		// If unique is nil, it wasn't passed with index definition.
```

Comment:
I'm not sure about this approach -- if we should set it all, or we just shouldn't store `unique` field in this case, or we should set it as `false`. But I understand that it could cause some difference in how FerretDB responds vs how MongoDB responds (e.g. FerretDB responds with `false`, and MongoDB with `nil`), so I'm ok to have it like this for now.